library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
-                        ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-PLMS <nowiki>[Periodic limb movements in sleep.]</nowiki>
-            *** EMG-spasm
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-tonic-contraction
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-asymmetric-activation 
-              **** EMG-asymmetric-activation-left-first
-                  ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-              **** EMG-asymmetric-activation-right-first
-                  ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Other-EMG-features {requireChild}
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * Other-polygraphic-channel {requireChild}
+        ** Myoclonus <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** Negative-myoclonus <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** Myoclonus-rhythmic <nowiki> [Numerical value for frequency. (Source: Beniczky ea 2017, Table 16.)] </nowiki>
+                *** <nowiki># {takesValue, valueClass=numericClass} </nowiki>
+            *** Myoclonus-arrhythmic <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** Myoclonus-synchronous <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** Myoclonus-asynchronous <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+        ** PLMS <nowiki>[Periodic limb movements in sleep. (Source: Beniczky ea 2017, Table 16.)]</nowiki>
+        ** Spasm <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+        ** Tonic-contraction <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+        ** Other-EMG-features <nowiki> {requireChild} [Source: Beniczky ea 2017, Table 16.]</nowiki>
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+    * Other-polygraphic-channel <nowiki> {requireChild}[Source: Beniczky ea 2017, Section 13.] </nowiki>
         ** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
 
 
 '''Finding-property''' <nowiki>[Descriptive element similar to main HED /Property. Something that pertains to a thing. A characteristic of some entity. A quality or feature regarded as a characteristic or inherent part of someone or something. HED attributes are adjectives or adverbs.]</nowiki>
 
-    * Signal-morphology-property <nowiki>[Signal morphology pertains to the visual and occasionally structural attributes employed for the description and characterization of findings.]</nowiki>
+    * Signal-morphology-property <nowiki>[Signal morphology contains attributes to describe findings in background, interictal or ictal activity. (Source: Beniczky ea 2017, Table 5, Table 8, Table 12.)]</nowiki>
```

Comment:
Maybe the following would be clearer?
"Signal morphology attributes relevant to  findings of background, interictal or ictal activity."

Reply:
fixed